packages/node/src/integrations/http/index.ts

```diff
@@ -45,8 +45,8 @@ interface HttpOptions {
   ignoreOutgoingRequests?: (url: string, request: RequestOptions) => boolean;
 
   /**
-   * Do not capture spans or breadcrumbs for incoming HTTP requests to URLs where the given callback returns `true`.
-   * This controls both span & breadcrumb creation - spans will be non recording if tracing is disabled.
+   * Do not capture spans for incoming HTTP requests to URLs where the given callback returns `true`.
+   * Spans will be non recording if tracing is disabled.
```

Comment:
I also adjusted the JSDoc f `ignoreIncomingRequests` because it wrongfully claimed that incoming requests also generate breadcrumbs. Which isn't the case now and wasn't the case before: https://github.com/getsentry/sentry-javascript/blob/d7749aeee11bc54dbba7f801b48655ebf48ad5e1/packages/node/src/integrations/http.ts#L245-L248